moped-editor/src/utils/mapHelpers.js

```diff
@@ -113,13 +168,19 @@ export const getInteractiveIds = () =>
   Object.values(mapConfig.layerConfigs).map(config => config.layerIdName);
 
 /**
- * Get a feature's ID attribute from a Mapbox map click or hover event
- * @param {Object} e - Event object for click or hover on map
- * @param {String} idKey - Key that exposes the id of the polygon in the layer
+ * Get the layer names from the layerConfigs object
+ * @return {Array} List of layer names set in mapConfig.layerConfigs
+ */
+export const getLayerNames = () => Object.keys(mapConfig.layerConfigs);
+
+/**
+ * Get a feature's ID attribute from a GeoJSON feature
+ * @param {Object} feature - GeoJSON feature taken from a Mapbox click or hover event
+ * @param {String} layerName - Name of layer to find lodash get path from layer config
  * @return {String} The ID of the polygon clicked or hovered
  */
-export const getFeatureId = (e, idKey) =>
-  e.features && e.features.length > 0 && e.features[0].properties[idKey];
+export const getFeatureId = (feature, layerName) =>
+  get(feature, mapConfig.layerConfigs[layerName].layerIdGetPath);
```

Comment:
👍

Reply:
Thanks for introducing lodash .get() into Moped because it was perfect for this config. 🙏